cardano-ping/cardano-ping.cabal

```diff
@@ -0,0 +1,64 @@
+cabal-version:        2.2
+name:                 cardano-ping
+version:              0.1.0.0
+synopsis:             Utility for pinging cardano nodes
+license:              Apache-2.0
+license-files:        LICENSE
+                      NOTICE
+copyright:            2019-2022 Input Output (Hong Kong) Ltd.
+author:               Duncan Coutts, Marc Fontaine, Karl Knutsson, Marcin Szamotulski, Alexander Vieth, Neil Davies, John Ky
+maintainer:           duncan@well-typed.com, marcin.szamotulski@iohk.io, marc.fontaine@iohk.io, karl.knutsson@iohk.io, alex@well-typed.com, neil.davies@pnsol.com, john.ky@iohk.io
+category:             Network
+build-type:           Simple
+extra-source-files:   CHANGELOG.md
+
+flag asserts
+  description:        Enable assertions
+  manual:             False
+  default:            False
+
+library
+  hs-source-dirs:       src
+  exposed-modules:      Network.Ping
+  build-depends:        base                          >= 4.9      && < 4.17,
+                        aeson                         >= 2.1.1.0  && < 3,
+                        cborg                         >= 0.2.8    && < 0.3,
+                        array                         >= 0.5      && < 0.6,
+                        binary                        >= 0.8      && < 0.11,
+                        bytestring                    >= 0.10     && < 0.12,
+                        containers                    >= 0.5      && < 0.7,
+                        contra-tracer                 >= 0.1      && < 0.2,
+                        io-classes                    >= 0.3      && < 0.4,
+                        monoidal-synchronisation      >= 0.1      && < 0.2,
+                        network-mux                   >= 0.2      && < 0.3,
+                        process                       >= 1.6      && < 1.7,
+                        quiet                         >= 0.2      && < 0.3,
+                        statistics-linreg             >= 0.3      && < 0.4,
+                        strict-stm                    >= 0.2      && < 0.3,
+                        tdigest                       >= 0.2.1.1  && < 0.3,
+                        text                          >= 1.2.4    && < 2.1,
+                        time                          >= 1.9.1    && < 1.14,
+                        vector                        >= 0.12     && < 0.13,
+
+                        -- -- -- The Windows version of network-3.1.2 is missing
+                        -- -- -- functions, see
+                        -- -- -- https://github.com/haskell/network/issues/484
```

Comment:
```suggestion
                        -- The Windows version of network-3.1.2 is missing
                        -- functions, see
                        -- https://github.com/haskell/network/issues/484
```

Reply:
_note to myself_: this issue is already fixed in the latest version, we should consider upgrading to a newer `network` release.